### ISSUE TITLE
Internal Error 500 - When Steam is down.

### ISSUE THREAD

**Issue by emilkaaekudahl** [ztizzlegaming/CSGOWinBig-SteamBot#15]
When steam is down it gives a internal error 500, and it does not do the thing it should (accept / send offer.) how can I make it do it after the crash? because the bot auto restarts?

**Comment by cover77** [comment]
I am not sure but isn't it because you have steam guard?

**Comment by emilkaaekudahl** [comment]
No, it happens only when Steam is unstable, i want it to catch errors and retry

**Comment by iLogix** [comment]
it is because steam is down... how could steam send and accept offers when its down. ERROR 500 always steam problem
